ext/console/02_console.js

```diff
@@ -257,7 +253,7 @@ function cliTable(head, columns) {
       const width = columnWidths[i] || 0;
       const counted = getStringWidth(value);
       columnWidths[i] = MathMax(width, counted);
-      columnRightAlign[i] &= canRightAlign(value);
+      columnRightAlign[i] &= NumberIsInteger(value);
```

Comment:
This doesn't work because value is already stringified here https://github.com/denoland/deno/blob/0597499e9ea09bb89409265b31e39baed2733a34/ext/console/02_console.js#L2157 (`value` always has type of 'string')

The below would be a potential fix
```suggestion
      columnRightAlign[i] &= NumberIsInteger(+value);
```

Reply:
Ah, nice, thanks! :smiley: